src/library/scala/collection/parallel/ParIterableLike.scala

```diff
@@ -1284,7 +1282,7 @@ self: ParIterableLike[T, Repr, Sequential] =>
   extends Transformer[Combiner[(U, S), That], Zip[U, S, That]] {
     @volatile var result: Result = null
     def leaf(prev: Option[Result]) = result = pit.zip2combiner[U, S, That](othpit, pbf())
-    protected[this] def newSubtask(p: IterableSplitter[T]) = unsupported
+    protected[this] def newSubtask(p: IterableSplitter[T]) = throw new UnsupportedOperationException
```

Comment:
The problem was that about half of the code in parallel used `unsupported`, while the other half used `throw new UnsupportedOperationException` with no particular reason I could uncover. It wasn't used anywhere else.

It wasn't in line with the style rules, so instead I just got rid of it completely.